test/03_USM.test.js

```diff
@@ -85,54 +85,60 @@ contract('USM', (accounts) => {
         it("doesn't allow minting with less than MIN_ETH_AMOUNT", async () => {
           const MIN_ETH_AMOUNT = await usm.MIN_ETH_AMOUNT()
           // TODO: assert MIN_ETH_AMOUNT > 0
-          await expectRevert(usm.mint(MIN_ETH_AMOUNT.sub(new BN('1')), { from: user }), '0.001 ETH minimum')
+          await expectRevert(usm.mint(user1, user2, MIN_ETH_AMOUNT.sub(new BN('1')), { from: user1 }), '0.001 ETH minimum')
         })
 
         describe('with existing USM supply', () => {
           beforeEach(async () => {
-            await usm.mint(oneEth, { from: user })
+            await usm.mint(user1, user1, oneEth, { from: user1 })
           })
 
           it('allows burning FUM', async () => {
-            const fumBuyPrice = (await usm.fumPrice(sides.BUY)).toString()
-            const fumSellPrice = (await usm.fumPrice(sides.SELL)).toString()
+            const fumBuyPrice = (await usm.fumPrice(sides.BUY))
+            const fumSellPrice = (await usm.fumPrice(sides.SELL))
 
-            const fumBalance = (await fum.balanceOf(user)).toString()
+            const fumBalance = (await fum.balanceOf(user1)).toString()
             const targetFumBalance = oneEth.mul(priceWAD).div(WAD) // see "allows minting FUM" above
```

Comment:
Maybe for consistency/clarity we should just never do `priceOrWhatev = x.toString(); x.should.equal(y.toString())`, and always do `priceOrWhatev = x; x.toString().should.equal(y.toString())`?

Reply:
(Don't feel super strongly about this)

Reply:
Fixed, values should be BigNumber (or BN) for the whole test, and converted to string only for and at the assertion.

Reply:
Nice!  Thank you.